### DISCUSSION TITLE
Open Daily Note hotkey

### DISCUSSION THREAD

**Discussion by KuSi833** [liamcain/obsidian-calendar-plugin#203] [Q&A]
I'm wondering if the Calendar plugin could add a command to open/create today's daily note, with an option in the settings to open it in a new tab or not.

I often need to open the daily note and opening the calendar and ctrl left clicking today's date becomes a hassle. The current command to open today's daily note is managed by the core Daily Note plugin and it both always opens the note in the current tab, and doesn't understand the Calendar plugin Template Tags.

**Comment by liamcain** [comment]
This is possible using the Periodic Notes plugin: https://github.com/liamcain/obsidian-periodic-notes

You can add a keybinding to "Periodic Notes: Open daily note"

![image](https://user-images.githubusercontent.com/693981/147687532-a24e0e4a-083d-4758-878f-96fc3b7d4368.png)

**Reply by KuSi833** [reply]
Sure, but is there a way to make it open in a new tab?
